### PR TITLE
Probot support

### PR DESCRIPTION
**Pull request by trieloff**
Adding support for Probot.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -64,6 +78,9 @@ async function report(checks = {}, timeout = 10000, decorator = { body: xml, mim
         resolveWithFullResponse: true,
         time: true,
         timeout,
+        headers: {
+          'user-agent': `Helix Status on behalf of ${name} v${version}`,
```
Comment (tripodsan) [inline src/index.js]: I would follow common syntax:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#Syntax

eg: 
`helix-status/1.0 (${name}@${version})`

File: src/index.js [hunk 2]
```diff
@@ -30,32 +31,45 @@ function xml(o, name) {
   return `<${name}>${value}</${name}>`;
 }
 
-async function getVersion() {
-  if (!_version) {
-    _version = await new Promise((resolve) => {
-      fs.readFile('package.json', 'utf-8', (err, data) => {
-        if (err) {
+function getPackage() {
```
Comment (tripodsan) [inline src/index.js]: I would prevent the package to be read twice (once for name, once for version)

**Comment by tripodsan** [comment]
not related to this issue - but can we remove the XML response and drop the xml-escape dependency?

**Comment by trieloff** [comment]
We can drop it after we don't use Pingdom anywhere anymore.

File: src/index.js [hunk 3]
```diff
@@ -64,6 +65,9 @@ async function report(checks = {}, timeout = 10000, decorator = { body: xml, mim
         resolveWithFullResponse: true,
         time: true,
         timeout,
+        headers: {
+          'user-agent': `helix-status/${version} (${name}@${pkgversion})`,
```
Comment (tripodsan) [inline src/index.js]: the wrong way around :-)

Comment (tripodsan) [reply src/index.js]: can you add test for correct user-agent ?

**Pull request merged**